CONVENTIONS.md

```diff
@@ -116,12 +116,85 @@ Name screen coordinates `x` and `y`, rather than `left` and `top`.
 
 ## Javadocs
 
-Write sentences for class, method and fields javadocs, starting with an uppercase and ending with a period. Start method docs with verbs, like `Gets` or `Called`. Use HTML tags such as `<p>` if the docs have several paragraphs.
+Write sentences for class, method and fields javadocs, starting with an uppercase and ending with a period. Start method docs with verbs, like `Gets` or `Called`. Use HTML tags such as `<p>` if the docs have several paragraphs, as line wraps are converted to spaces in the generated documentation. Feel free to start a new line whenever you feel the current line is too long.
 
-Write quick descriptions for parameter javadocs as well as `@return` tags, with no uppercase or period. Add parameter docs to the parameter itself instead of using the `@param` tag.
+Parameter and `@return` documentation should use quick descriptions without initial capitalization or punctuation, such as `whether the block placement was successful`.
+
+Avoid using the `@param` tag for documentation of owner methods. Add parameter docs to the parameter itself instead of adding `@param` tags to its owner method. However, you can use the tag for type parameters (such as `<T>` in `public class Lazy<T>`), who don't have mapping entries for their documentations.
```

Comment:
```suggestion
Avoid using the `@param` tag for documentation of owner methods. Add parameter docs to the parameter itself instead of adding `@param` tags to its owner method. However, you can use the tag for type parameters (such as `<T>` in `public class Lazy<T>`), which do not support separate documentation.
```

Reply:
```suggestion
Avoid using the `@param` tag for documentation of owner methods. Add parameter docs to the parameter itself instead of adding `@param` tags to its owner method. However, you can use the tag for type parameters (such as `<T>` in `public class Lazy<T>`), which cannot be documented separately.
```
Imo this one better